### DISCUSSION TITLE
Add Version number to the name of the executable files on each release.

### DISCUSSION THREAD

**Discussion by shobeira** [tutao/tutanota#6694] [Ideas]
This way, there is no need to run the executable in order to see if it is the latest version. Alternatively, add date of release to the executable files instead of version number.

**Comment by charlag** [comment]
It doesn't work with desktop shortcuts and such, does it?

**Reply by shobeira** [reply]
Here is the root of the issue:

Althogh the AppImage get updated automatically, imagine several executable (e.g: .AppImage file for linux) is downloaded over the time.

By having the app version embedded into the executable name (rather than finding the download date of the file), It would be much easier to know the initial executable version rather than running the executable and find it out. Or in case of having a problem with the recent version, you need to download an older version and when the name is same, the confusion happens.
